src/components/AssetsFilters/AssetsFilters.scss

```diff
@@ -1,4 +1,9 @@
-ul.filter-set {
-  list-style: none;
-  padding-left: $spacer;
+@import "~bootstrap/scss/utilities/_spacing.scss";
+
+.filter-heading {
+	@extend .pt-3;
```

Comment:
This indentation looks a little funny relative to the indentation below. Should this be two spaces instead of what looks like a tab?

Reply:
Good eye :)